setup/pedalboards/get_pedalboards.sh

```diff
@@ -15,12 +15,12 @@
 # You should have received a copy of the GNU General Public License
```

Comment:
"code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

Reply:
the license is already there